src/database/migrations/V1__addTest.ts

```diff
@@ -1,21 +1,28 @@
 import knex from 'knex';
 
 const TEST_TABLE = 'test';
+const TEST_RESULTS_TABLE = 'test_results';
 
 export async function up(db: knex) {
+  await db.schema.createTable(TEST_TABLE, (table) => {
+    table.uuid('id').primary();
+    table.uuid('user_id').notNullable();
+    table.uuid('test_type_id').notNullable();
+    table.timestamp('creation_time').notNullable().index();
+  });
+
   await db.schema
-    .createTable(TEST_TABLE, (table) => {
+    .createTable(TEST_RESULTS_TABLE, (table) => {
       table.uuid('id').primary();
-      table.uuid('user_id').notNullable();
-      table.uuid('test_type_id').notNullable();
-      table.timestamp('creation_time').notNullable().index();
-      table.jsonb('results').nullable();
-      table.uuid('results_creator_id').nullable();
-      table.timestamp('results_creation_time').nullable();
+      table.uuid('test_id').unique();
+      table.uuid('creator_id').notNullable();
+      table.jsonb('details').nullable();
+      table.timestamp('creation_time').notNullable();
     })
-    .raw('CREATE INDEX IDX_RESULTS on "test" USING GIN (results)');
+    .raw('CREATE INDEX IDX_RESULTS on "test_results" USING GIN (details)');
```

Comment:
I know this was here before, but do we need it?

Reply:
We need it if we want to quickly be able to query based on reslults properties. 
i.e. `select all results where details.c = 'true' OR details.igg = 'false'`
It is a full text search index for JSONB